pyfar/classes/audio.py

```diff
@@ -1011,6 +1025,41 @@ def power(data: tuple, domain='freq'):
     return _arithmetic(data, domain, _power)
 
 
+def matrix_multiplication(data: tuple, domain='freq'):
+    """Matrix multiplication along first two axes of pyfar audio objects
+    and array likes.
+
+    Pyfar audio objects are: :py:func:`Signal`, :py:func:`TimeData`, and
+    :py:func:`FrequencyData`.
+
+
+    Parameters
+    ----------
+    data : tuple of the form (data_1, data_2, ..., data_N)
+        Data to be multiplied. Can contain pyfar audio objects, array likes,
+        and scalars. Pyfar audio objects can not be mixed, e.g.,
+        :py:func:`TimeData` and :py:func:`FrequencyData` objects do not work
+        together. See
+        :py:mod:`arithmetic operations <pyfar._concepts.arithmetic_operations>`
+        for possible combinations of Signal FFT normalizations.
+    domain : ``'time'``, ``'freq'``, optional
+        Flag to indicate if the operation should be performed in the time or
+        frequency domain. If working in the frequency domain, the FFT
```

Comment:
Should we already change this to 'Frequency domain operations work on the raw spectrum' and also change this in the other docstrings?

Reply:
Mentioned it in issue #320. I'd suggest to solve in a separate PR for all arithmetic operations.